setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
```

Comment:
I haven't tested it. It's from a `cookiecutter-pypackage` template, so I'd expect it to work.